src/JBGovernanceNFT.sol

```diff
@@ -169,18 +130,18 @@ contract JBGovernanceNFT is ERC721Votes {
         // batchSize is used if inherited from `ERC721Consecutive`
         // which we don't, so this should always be 1
         assert(batchSize == 1);
-        uint256 _stakingTokenAmount = stakes[firstTokenId].amount;
-
-        // TODO: check if we can do this 'unchecked'
-        if (from != address(0)) {
-            stakingTokenBalance[from] -= _stakingTokenAmount;
-        }
-
-        if (to != address(0)) {
-            stakingTokenBalance[to] += _stakingTokenAmount;
+        
+        uint256 _stakeAmount = stakingTokenBalance[from];
```

Comment:
We should track the token amount for each token, if a user has 2 tokens and they transfer 1 of them this will transfer the entire staked balance.

Reply:
Pushed in the changes couple of days back 👍🏻

Reply:
@xBA5ED would you be able to review today ?